desktop/core/src/desktop/tests.py

```diff
@@ -1467,16 +1491,16 @@ def test_db_migrations_sqlite():
     file_name = 'hue_' + version + '.db'
     path = get_desktop_root('./core/src/desktop/test_data/' + file_name)
     DATABASES[name] = {
-      'ENGINE' : 'django.db.backends.sqlite3',
-      'NAME' : path,
-      'USER' : '',
-      'SCHEMA' : 'public',
-      'PASSWORD' : '',
-      'HOST' : '',
-      'PORT' : '',
-      'OPTIONS' : '',
-      'ATOMIC_REQUESTS' : True,
-      'CONN_MAX_AGE' : 0,
+      'ENGINE': 'django.db.backends.sqlite3',
+      'NAME': path,
+      'USER': '',
+      'SCHEMA': 'public',
+      'PASSWORD': '',
+      'HOST': '',
+      'PORT': '',
+      'OPTIONS': {},
```

Comment:
Will break with Python2?
(again, when we change some logic it will very probably have an impact on current code)

Reply:
Yes, I thought about it, but as it passing the unit test so I guess, it will not break the py2.